commands/wish.js

```diff
@@ -67,72 +76,90 @@ function validateArgs(args, message) {
     c: "character",
     w: "weapon",
     b: "beginner",
-    s: "standard"
-  }
+    s: "standard",
```

Comment:
I don't think there should be a comma here @accelthreat .

Reply:
Different linting tools, still valid js

Reply:
Okay cool. Just had to clarify since I'm not really well-versed to JS myself. We can keep it as is.😂